src/charm.py

```diff
@@ -1361,10 +1367,13 @@ def _is_workload_running(self) -> bool:
     def update_config(self, is_creating_backup: bool = False) -> bool:
         """Updates Patroni config file based on the existence of the TLS files."""
         enable_tls = all(self.tls.get_tls_files())
+        limit_memory = None
+        if self.config.profile_limit_memory:
+            limit_memory = self.config.profile_limit_memory * 10**6
```

Comment:
Should we use `1024 * 1024` instead of `10**6` as the config description states `MegaBytes`?

Reply:
I think MB to B is `10**6` or am I wrong?

Reply:
Does the docs specify if it's MB or MiB?

Reply:
Spec is MB, and pg config knows how to handle different suffixes. Charm calculations being done in bytes